crowdsale/crowdsale.json

```
@@ -4,7 +4,7 @@
     "keyStore": null,
     "from": "hxaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
     "to": "cx0000000000000000000000000000000000000000",
-    "stepLimit": "0x77359400",
+    "stepLimit": "0x40000000",
```

From my experiment, it took 0x4040b3a8 in case of crowdsale deployment.
So it should be increased over "0x40000000" like the following.
```suggestion
    "stepLimit": "0x50000000",
```